migrations/schema.sql

```diff
@@ -1,8 +1,8 @@
 -- MySQL dump 10.13  Distrib 5.7.23, for Linux (x86_64)
 --
--- Host: localhost    Database: cpjudge
+-- Host: localhost    Database: gocoder_dev
 -- ------------------------------------------------------
--- Server version	5.7.23-0ubuntu0.18.04.1
+-- Server version	5.7.23-0ubuntu0.16.04.1
```

Comment:
```suggestion
-- Server version	5.7.23-0ubuntu0.18.04.1
```
Don't change this file in this PR. We'll add to .gitignore